### PR TITLE
Update CircuitPython Essentials Pin Map Script for Pico W

### PR DESCRIPTION
**Pull request by anecdata**
Also add a commented line to show `microcontroller` pins.

Requested by a user on Discord: https://discord.com/channels/327254708534116352/537365702651150357/1074369255941877921

### CODE REVIEW

File: CircuitPython_Essentials/Pin_Map_Script/code.py [hunk 1]
```diff
@@ -1,15 +1,22 @@
...
+try:
+    import cyw43  # raspberrypi
+except ImportError:
+    cyw43 = None
 
 board_pins = []
 for pin in dir(microcontroller.pin):
-    if isinstance(getattr(microcontroller.pin, pin), microcontroller.Pin):
+    if isinstance(getattr(microcontroller.pin, pin), microcontroller.Pin) or \
+       (cyw43 and isinstance(getattr(microcontroller.pin, pin), cyw43.CywPin)):
```
Comment (dhalbert) [inline CircuitPython_Essentials/Pin_Map_Script/code.py:16]:
```suggestion
    if (isinstance(getattr(microcontroller.pin, pin), microcontroller.Pin) or 
        (cyw43 and isinstance(getattr(microcontroller.pin, pin), cyw43.CywPin))):
```

**Review by dhalbert — CHANGES_REQUESTED** [review]
Just a formatting change.

File: CircuitPython_Essentials/Pin_Map_Script/code.py [hunk 2]
```diff
@@ -1,15 +1,22 @@
...
 import board
+try:
+    import cyw43  # raspberrypi
+except ImportError:
+    cyw43 = None
 
 board_pins = []
 for pin in dir(microcontroller.pin):
-    if isinstance(getattr(microcontroller.pin, pin), microcontroller.Pin):
+    if (isinstance(getattr(microcontroller.pin, pin), microcontroller.Pin) or
```
Comment (dhalbert) [inline CircuitPython_Essentials/Pin_Map_Script/code.py:15]:
```suggestion
    if (isinstance(getattr(microcontroller.pin, pin), microcontroller.Pin) or
```

**Review by dhalbert — CHANGES_REQUESTED** [review]

**Comment by anecdata** [comment]
Is there more to do here? Dan made the changes and it's better now. Change requests appear to be resolved, but Github still doesn't look happy.

**Comment by anecdata** [comment]
fixed conflict, and removed commented-out lines to show microcontroller pins since that was redundant with the added lies to show original GPIO parenthetically

(history of this little code snippet, as close as I could reconstruct it, is [here](https://gist.github.com/anecdata/1c345cb2d137776d76b97a5d5678dc97))

**Review by dhalbert — APPROVED** [review]
Looks good - thanks!

**Pull request merged**